### PR TITLE
Cleanup animation docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/animation.py [hunk 1]
```diff
@@ -291,21 +291,19 @@ def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
...
-            the output file size and quality. The default value is ``None``,
-            which uses :rc:`animation.bitrate`.  A value of -1 implies that
-            the bitrate should be determined automatically by the underlying
-            utility.
+            Movie frame rate (per second).
+        codec : str or None, default: :rc:`animation.codec`
+            The codec to use.
+        bitrate : int, default: :rc:`animation.bitrate`
+            The number of kilobits used per second in the compressed movie, in
+            kilobits per second.  Higher values means higher quality movies,
```
Comment (timhoffm) [inline lib/matplotlib/animation.py]: Kilobits per second is duplicate.

File: lib/matplotlib/animation.py [hunk 2]
```diff
@@ -978,50 +974,48 @@ def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
...
+            Controls the dots per inch for the movie frames.  Together with
+            the figure's size in inches, this controls the size of the movie.
+
+        codec : str, optional, default: :rc:`animation.codec`.
+            The video codec to use.  Not all codecs are supported by a given
+            `MovieWriter`.
+
+        bitrate : int, default: :rc:`animation.bitrate`
+            The number of kilobits used per second in the compressed movie, in
+            kilobits per second.  Higher values means higher quality movies,
```
Comment (timhoffm) [inline lib/matplotlib/animation.py]: Duplicate as above

File: lib/matplotlib/animation.py [hunk 3]
```diff
@@ -291,21 +291,19 @@ def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
         Parameters
         ----------
         fps : int
```
Comment (timhoffm) [inline lib/matplotlib/animation.py]:
```suggestion
        fps : int, default: 5
```

File: lib/matplotlib/animation.py [hunk 4]
```diff
@@ -978,50 +974,48 @@ def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
...
+        writer : `MovieWriter` or str, default: :rc:`animation.writer`
             A `MovieWriter` instance to use or a key that identifies a
-            class to use, such as 'ffmpeg'. If ``None``, defaults to
-            :rc:`animation.writer` = 'ffmpeg'.
+            class to use, such as 'ffmpeg'.
 
-        fps : number, optional
-           Frames per second in the movie. Defaults to ``None``, which will use
-           the animation's specified interval to set the frames per second.
+        fps : int or None, optional
```
Comment (timhoffm) [inline lib/matplotlib/animation.py]: None can be implicit

> If not given, the frame rate is derived from ...

**Comment by anntzer** [comment]
thanks, all agreed.

**Pull request merged**